iron-overlay-behavior.html

```diff
@@ -145,6 +145,14 @@
       },
 
       /**
+       * Is true during the transition from close to open and open to close.
+       */
+      transitioning: {
```

Comment:
`IronOverlayManager` checks for the `transitioning` property [since ever](https://github.com/PolymerElements/iron-overlay-behavior/commit/09e757a14cd6cab9ba13d961ef38d12f36b57359#diff-3a555b03d77a3dc28aad3488eed08943R73) in order to apply or not the focus, but I can probably remove this and keep the `__isAnimating` property as before.